samples/local-service-registry/local-registry-client/src/main/java/org/apache/servicecomb/samples/localregistry/localregistryclient/LocalRegistryClient.java

```diff
@@ -30,10 +30,14 @@
   private static RestTemplate templateNew = RestTemplateBuilder.create();
 
   public static void main(String[] args) throws Exception {
-    System.setProperty("local.registry.file", "src/main/resources/registry.yaml");
+    String absoluteFilePath = ClassLoader.getSystemResource("").getPath() + "registry.yaml";
```

Comment:
after package to jar, can not load the file
to support this feature, should modify org.apache.servicecomb.serviceregistry.client.LocalServiceRegistryClientImpl#LocalServiceRegistryClientImpl()?

Reply:
I try to modify LocalServiceRegistryClientImpl class to check whether the file exists and read the file by classloader, the file must be put in the resource directory. Test is OK, pls help me to review,thks